docs/examples.md

```diff
@@ -197,7 +197,8 @@ CREATE TABLE pageviews_per_region_per_30secs AS \
 ```
 
 As you can see we used UCASE and LCASE functions in KSQL to convert the values of gender and
```

Comment:
`UCASE and LCASE functions in KSQL are used to convert the values of gender and regionid columns to upper and lower case, so that you can match them correctly.` (remove `As you can see` and `respectively`)